sdk/formrecognizer/ai-form-recognizer/samples/v3/typescript/src/recognizeIdentityDocument.ts

```diff
@@ -36,7 +36,7 @@ export async function main() {
   const readStream = fs.createReadStream(fileName);
 
   const client = new FormRecognizerClient(endpoint, new AzureKeyCredential(apiKey));
-  const poller = await client.beginRecognizeIdDocuments(readStream, {
+  const poller = await client.beginRecognizeIdentityDocuments(readStream, {
```

Comment:
I can't comment on it, but make sure you update `printField("Country");` in all samples